### PR TITLE
Add Brain Team section

### PR DESCRIPTION
**Pull request by trickeydan**
Closes https://github.com/srobo/brain-team-tasks/issues/5

Added Kit Committee for review just for structure of the kitbook purposes.

### CODE REVIEW

**Review by rgilton — APPROVED** [review]
:+1:

File: docs/Brain-Team/2021-Software/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Goals
+
+The 2021 Kit software aims to replace the 2015 software, with a launch targeted for SR2021 Kickstart.
```
Comment (keysmashes) [inline docs/Brain-Team/2021-Software/index.md]: SR2021 is the virtual competition that's just been launched, we're targeting SR2022 afaik.

Comment (trickeydan) [reply docs/Brain-Team/2021-Software/index.md]: Good point. I had in my head that SR2021 was in 2022 for some reason.

Comment (trickeydan) [reply docs/Brain-Team/2021-Software/index.md]: b5b4957

**Review by keysmashes — APPROVED** [review]

**Pull request merged**